doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+
+## Valid transitions
+
+### Create entity with `dateExpires` attribute
+
+An entity is created with transient nature if it includes the `dateExpires` attribute. For instance:
+
+```
+POST /v2/entities
+{
+  "id": "t1",
+  "type": "Ticket",
+  ...
+  "dateExpires": {
+    "value": "2028-07-07T21:35:00Z",
+    "type": "DateTime"
+  }
+}
+```
+
+creates an entity that will expire at July 7th, 2028 at 21:35.
+
+Additional considerations:
+
+* `dateExpires` has to have a valid `DateTime` value (check NGSIv2 specification for details).
+Otherwise a 400 Bad Request would be returned.
+
+* If `dateExpires` is set in the past, the entity is created expired (a bit weird, but functionally correct).
+
+### Add `dateExpires` attribute to entity that previously doesn't have it
+
+We can add the `dateExpires` attribute to a regular entity (e.g. "t2"). For instance:
+
+```
+POST /v2/entities/t2/attrs
+{
+  "dateExpires": {
+    "value": "2028-10-12T14:23:00Z",
+    "type": "DateTime"
+  }
+}
+```
+
+will make that entity to expire on October 12th, 2028 at 14:23.
+
+Additional considerations:
+
+* `dateExpires` has to have a valid `DateTime` syntax (check NGSIv2 specification for details).
+Otherwise a 400 Bad Request would be returned.
+
+* If `dateExpires` is set in the past, the entity gets automatically expired.
+
+### Update `dateExpires` attribute in entity that previously has it
+
+Context Broker allows several ways of updating an attribute value (check NGSIv2 specification for
+details). For instance using PUT on the attribute resource URL:
+
+```
+PUT /v2/entities/t2/attrs/dateExpires
+{  
+  "value": "2028-12-31T23:59:00Z",
+  "type": "DateTime"
+}
+```
+
+will change expiration date to December 31th, 2028 at 23:59.
+
+Additional considerations:
+
+* `dateExpires` has to have a valid `DateTime` syntax (check NGSIv2 specification for details).
+Otherwise a 400 Bad Request would be returned.
+
+* If `dateExpires` is set in the past, the entity gets automatically expired
+
+### Remove `dateExpired`attribute from entity
+
+Finally, you can remove `dateExpires` attribute from a transient entity:
+
+```
+DELETE /v2/entities/t2/attrs/dateExpires
+```
+
+This become the entity into a regular (i.e. not transient) entity and will not be deleted due to expiration. 
+
+## Deletion of expired entities
+
+Expiration relies on the MongoDB feture to [expire documents at a specific clock time](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-at-a-specific-clock-time). This is based in a background thread thats wakes up every 60 seconds,
+so your transient entities may remain in the database up to 60 seconds (or a bit more, if the MongoDB load is high) after
+expiration date (see [MongoDB documentation](https://docs.mongodb.com/manual/core/index-ttl/#timing-of-the-delete-operation)
+for more details).
+
+**Once a transient entity is removed, it cannot be recovered.**
+
+## Backward compatibility considerations
```

Comment:
This is based in my "theoretical" interpretation of the implementation in #3173 . However, I may be wrong. This should be tested "on real" (once #3173) gets merged but on the menawhile let's try this description to be as precise as possible.

@arigliano what do you think? Is the description on this section correct?

Reply:
Yes, It's OK 👍

Reply:
Thanks for checking it! NTC